packages/@expo/cli/src/start/server/middleware/metroOptions.ts

```diff
@@ -46,11 +46,13 @@ export type ExpoMetroOptions = {
   runModule?: boolean;
 };
 
+// See: @expo/metro-config/src/serializer/fork/baseJSBundle.ts `ExpoSerializerOptions`
 export type SerializerOptions = {
```

Comment:
Maybe it's better to pull in the `ExpoSerializerOptions` from `@expo/metro-config` directly?